docs/subgraph-manifest.md

```diff
@@ -34,6 +34,7 @@ Any data format that has a well-defined 1:1 mapping with the [IPLD Canonical For
 | **network** | *String* | For blockchains, this describes which network the subgraph targets. For Ethereum, this could be, for example, "mainnet" or "rinkeby". |
 | **source** | [*EthereumContractSource*](#151-ethereumcontractsource) | The source data on a blockchain such as Ethereum. |
 | **mapping** | [*Mapping*](#152-mapping) | The transformation logic applied to the data prior to being indexed. |
+| **templates** | [*Dynamic Data Source Spec*](#17-dynamic-data-source) | Each dynamic data source spec defines a template of a traditional data source. This template will be created and mapped to a live data source at runtime. |
```

Comment:
Rather than "dynamic data source spec", could we call this `Data Source Templates`?

Also, could we make the description: "Each data source template defines a data source that can be created dynamically from the mappings."?